content/en/docs/instrumentation/python/automatic/logs-example.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Logs Auto-Instrumentation Example
+linkTitle: Logs Example
+weight: 20
+# prettier-ignore
+cSpell:ignore: distro instrumentor mkdir MSIE Referer Starlette uninstrumented virtualenv WSGI
+---
+
+This page demonstrates how to use Python logs auto-instrumentation in
+OpenTelemetry.
+
+Unlike Traces and Metrics, there is no Logs API. There is only an SDK. For
```

Comment:
Might be worth clarifying here that there *is* a logs bridge API, but it *is* different from the traces/metrics api in that it's not how application developers would be creating logs. Instead they would use this bridge API to setup log appenders in the standard logging libraries. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md

Reply:
Thank you! This is super helpful!!